source/extensions/transport_sockets/alts/config.cc

```diff
@@ -38,7 +38,8 @@ void grpcAltsSetRpcProtocolVersions(grpc_gcp_rpc_protocol_versions* rpc_versions
 // Returns true if the peer's service account is found in peers, otherwise
 // returns false and fills out err with an error message.
 bool doValidate(const tsi_peer& peer, const absl::node_hash_set<std::string>& peers,
-                std::string& err) {
+                TsiInfo& tsi_info, std::string& err) {
+  tsi_info.name_ = TSI_ALTS_SERVICE_ACCOUNT_PEER_PROPERTY;
```

Comment:
You should set it to the `value` variable down inside the the `if (name.compare(TSI_ALTS_SERVICE_ACCOUNT_PEER_PROPERTY) ... ` statement.